app/components/tables/holidays_table/table_element/PaginationBar.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {useState} from "react";
+
+interface paginationBarProps {
+    tableInstance : any
+}
+const PaginationBar = ({tableInstance} : paginationBarProps) => {
+    const [activePage, setActivePage] = useState(0);
+    const startPage = Math.max(0, activePage - 2);
+    const endPage = Math.min(tableInstance.getPageCount() - 1, startPage + 4);
```

Comment:
`startPage` will always be 0 since `activePage` is declared as 0 above it.  Also could you explain the logic of endPage? Thanks.

Reply:
The 'start' and 'end' page variables refer only to the first and last numbered pages that are currently displayed on the pagination bar.

The way this system is set up, the start page of the currently displayed pagination bar will be the current page - 2, after the first 5 pages. In addition, after the first 5 pages, the end page, which is the last page on the currently displayed pagination bar, is the start page plus the next 4 pages.

The reasoning behind this, is that I want to display only 5 numbered pages at a time in the pagination bar, allowing the user to see current page, a couple previous pages, and a couple of the next pages. The currently selected page will be in the center unless it's at the very front of the total pages or the very end.